packages/app-desktop/checkForUpdates.ts

```
@@ -65,7 +65,7 @@ async function addSkippedVersion(s: string) {
 }
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
-export default async function checkForUpdates(inBackground: boolean, parentWindow: any, options: CheckForUpdateOptions) {
+export default async function checkForUpdates(inBackground: boolean, parentWindow: any, options: CheckForUpdateOptions, dispatch?: (action: any)=> void) {
```

ESLint

[error] 'action' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)